yarn.lock

```diff
@@ -9816,12 +9821,51 @@ react-native-mail@3.0.7, react-native-mail@^3.0.5:
   resolved "https://registry.yarnpkg.com/react-native-mail/-/react-native-mail-3.0.7.tgz#07e6aca43f6f1f85f3e58e96e7fcd16b31a927b8"
   integrity sha512-FHe4kKJKAGuCPPWcDwaRSjdUhEE4IWIsVtTZ05dgD/MgOm00isYmM20zuJHla7oiHVQ2HmzGji76g6IuhIrCjw==
 
+react-native-material-buttons@^0.5.0:
```

Comment:
This does not appear to be used

Reply:
required by react-native-material-dropdown